execution_engine_testing/tests/src/test/contract_messages.rs

```diff
@@ -101,12 +99,12 @@ fn upgrade_messages_emitter_contract(
     )
     .build();
 
-    let new_topics = BTreeMap::from([(
-        MESSAGE_EMITTER_GENERIC_TOPIC.to_string(),
-        MessageTopicOperation::Add,
-    )]);
+    // let new_topics = BTreeMap::from([(
```

Comment:
Should we remove commented code?

Reply:
i'm less picky about it in test code...particularly when cleaning up other peoples code as this PR is doing...the original dev in their glorious SME-ness thought it useful to dump that data to stdout while working on the initial feature, and we may find it useful as well when we continue the work on it in their absence. its harmless, and we can always remove it later if it proves to be irrelevant.